deno/payloads/v9/channel.ts

```diff
@@ -79,49 +109,78 @@ export interface APIChannel extends APIPartialChannel {
 	 * The absence of this field in API calls and Gateway events should indicate that slowmode has been reset to the default value.
 	 */
 	rate_limit_per_user?: number;
+}
+export type APINewsChannel = APIGuildTextChannel<ChannelType.GuildNews>;
+export type APIGuildCategoryChannel = APIGuildChannel<ChannelType.GuildCategory>;
```

Comment:
Can't have nsfw

Reply:
Unless I'm reading it wrong, the docs show these can have `nsfw`:

https://discord.com/developers/docs/resources/channel

> Example News Category
```json
{
  "id": "41771983423143937",
  "guild_id": "41771983423143937",
  "name": "important-news",
  "type": 5,
  "position": 6,
  "permission_overwrites": [],
  "nsfw": true,
  "topic": "Rumors about Half Life 3",
  "last_message_id": "155117677105512449",
  "parent_id": "399942396007890945",
  "default_auto_archive_duration": 60
}
```

> Example Channel Category
```json
{
  "permission_overwrites": [],
  "name": "Test",
  "parent_id": null,
  "nsfw": false,
  "position": 0,
  "guild_id": "290926798629997250",
  "type": 4,
  "id": "399942396007890945"
}
```

Reply:
Right..okay, you can ignore this!